bitnami/oauth2-proxy/Chart.yaml

```
@@ -30,4 +30,4 @@ name: oauth2-proxy
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/oauth2-proxy
   - https://github.com/oauth2-proxy/oauth2-proxy
-version: 3.4.9
+version: 3.4.10
```

I think is a better upgrade to a minor since you are adding a new possibility.
```suggestion
version: 3.5.0
```